python/connectors/show_connector_status.py

```diff
@@ -0,0 +1,121 @@
+# Checks if connector has run in the last 24 hours.  If not starts it up.
+
+import os
+import sys
+import json
+import requests
+from datetime import datetime, timedelta
+
+# Returns connector information from the List Connectors API.
+def get_connectors(base_url, headers):
```

Comment:
Duplicated function? Could we make "helper functions" file to contain this?

Reply:
Yeah, good idea.